### ISSUE TITLE
Support request- Manual outdated? Grid. Unrecognized option "resource" under  "sylius_grid.grids.

### ISSUE THREAD

**Issue by Mr-Negative** [Sylius/Sylius#5742]
When i configure grid like this
http://docs.sylius.org/en/latest/bundles/SyliusGridBundle/your_first_grid.html
it gives me error

`Invalid type for path "sylius_grid.grids.app_blog_post.driver". Expected array, but got string`
when i add name: 

```
sylius_grid:
    grids:
        app_blog_post:
            driver:
                name: doctrine/orm
            resource: app.blog_post
```

but now it's showing:
`Unrecognized option "resource" under "sylius_grid.grids.app_blog_post"`

**Comment by TheMadeleine** [comment]
@pjedrzejewski @michalmarcinkowski Solved by #6312

**Comment by michalmarcinkowski** [comment]
@TheMadeleine thanks! 👍
